cmds/exp/pflask/pflask.go

```diff
@@ -347,17 +378,21 @@ func main() {
 
 	unprivileged := a[len(a)-1] == "#u"
 
-	// unlike pflask, we require that you set a chroot.
+	// unlike the original pflask, we require that you set a chroot.
 	// If you make it /, strange things are bound to happen.
 	// if that is too limiting we'll have to change this.
 	if *chroot == "" {
-		log.Fatalf("you are required to set the chroot via --chroot")
+		log.Fatalf("you are required to set the chroot via -chroot")
```

Comment:
The root argument should just become a required argument. Mistake on my part. I don't like required options ...